lib/processor.js

```diff
@@ -32,6 +32,12 @@
 			var pxre = /([\d\.]+)\D*/;
 			var width = svgelem.getAttribute( "width" );
 			var height = svgelem.getAttribute( "height" );
+			var viewbox = svgelem.getAttribute( 'viewBox' );
+			if (viewbox && width === '100%' && height === '100%') {
```

Comment:
Turns out I have actually suggested that in the past and pointed out a possible downside. [Please see my comment](https://github.com/filamentgroup/svg-to-png/issues/5#issuecomment-75586538).